pagopa/pn-frontend
# PN-5528 - link in domicile banner reachable through tab - clickable through keyboard

## Short description
Changed the DomicileBanner component so that the link contained is included in the tab navigation and can be launched through the Enter key.

## List of changes proposed in this pull request
Changed the MUI link to be rendered as a button (`component='button'`).

## How to test
Enter PF or PG with a user without domicile (e.g. ada for PF), go to the initial page, try out the tab navigation, it should include the link "Aggiungi PEC / Aggiungi telefono / Aggiungi mail / Attiva il servizio", and when the link is reached, pressing Enter should redirect to the contacts page.
